docs/sources/configure/integrations/references/slack/index.md

```diff
@@ -0,0 +1,382 @@
+| usergroups:write | Create and manage user groups                                                    | Required for connecting on-call schedules to Slack user groups                                                                       |
+| usergroups:read  | View user groups in a workspace                                                  | Required for connecting on-call schedules to Slack user groups                                                                       |
+
+### Message and conversation access
+
+| Permission           | Description                                                                                                        | Purpose                                                                                                                                            |
+|----------------------|--------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| app_mentions:read    | View messages and other content in public channels that Grafana IRM is a part of                                   | Enables the app to read messages and related content within authorized channels                                                                    |
+| chat:write           | Post messages in approved channels & conversations                                                                 | Allows @GrafanaIRM to post messages to Slack                                                                                                       |
+| chat:write.customize |  Send messages as Grafana IRM with a customized username and avatar                                                | Allows @GrafanaIRM to post messages to Slack                                                                                                       |
+| chat:write.public    | Send messages to channels GrafanaI RM isn't a member of                                                            | Allows @Grafan IRM to post messages to Slack                                                                                                       |
+| reaction:read        | View emoji reactions and their associated content in channels and conversations that Grafana IRM has been added to | Allows @GrafanaIRM to monitor message events for the 🤖 emoji to be included in the incident timeline                                               |
+| reaction:write       | Add and edit emoji reactions                                                                                       | This permission is required for resolution notes. More specifically – to include messages with the 📝emoji on resolution notes                      |
```

Comment:
My bad here ' More specifically – to include messages with the 📝emoji on resolution notes' should be to "mark resolution notes messages"